inductiva/coastal/README.md

```diff
@@ -8,83 +8,61 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
+Note: You can follow this example in a notebook or run the script `test.py`
+available in this folder, that contains all the code snippets exhibited here.
+
 Start by creating or loading a bathymetry. This can be done e.g. by reading an
 [ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
 a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
-Here we shall use the bathymetry data from Algarve available [here](https://sextant.ifremer.fr/record/SDN_CPRD_590_HR_Lidar_Algarve/)
-as an example.
+Here we shall use a artificial bathymetry, randomly generated by our code.
 
+We start by generating a random bathymetry.
 ```python
 import inductiva
 
-bathymetry = inductiva.coastal.Bathymety.from_ascii_xyz_file(
-    ascii_xyz_file_path="590_HR_Lidar_Algarve.emo")
-```
-
-The bathymetry can be inspected by plotting it with a given resolution, in this
-case 200 m x 200 m:
-
-```python
-bathymetry.plot(x_resolution=200, y_resolution=200)
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(
+        x_range=(0, 100),
+        y_range=(0, 100),
+        x_num=100,
+        y_num=100,
+        max_depth=50)
```

Comment:
I just saw the png for this bathymetry. It's too flat :) Can you pls increase the roughness / noise that it become more epic?

Reply:
About this: I generated bathymetries with higher roughness factors, and the simulations "exploded" - the water levels went to unphysical values (>500 m).
While I dont understand the source of this error I am not confortable in increasing the roughness factor in the example